.github/workflows/release.yml

```
@@ -28,17 +28,49 @@ jobs:
           node-version-file: '.nvmrc'
           cache: 'npm'
 
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: magefiles/go.mod
+
       - name: Set git 'user.name' and 'user.email'
         run: |
           git config user.name "${{ env.git-user }}"
           git config user.email ${{ env.git-email }}
 
+      - name: Retrieve latest Botkube release tag
+        run: |
+
+
       - name: Create a release branch
         run: |
           git checkout -b release/${{ inputs.version }}
           git push --set-upstream origin release/${{ inputs.version }}
           npm install
+
+      - name: Update versions in installation docs
+        run: |
+          LATEST_RELEASE_VERSION=$(curl -s -XGET  https://api.github.com/repos/kubeshop/botkube/releases/latest | awk -F'"tag_name": ' '{ print $2 }' | tr -d "\n" | awk -F'"' '{ print $2 }')
+          PREVIOUS_RELEASE_VERSION=$(grep  "\--version" docs/installation/slack/index.md | head -1 | awk -F'--version ' '{print $2}' | awk -F' ' '{print $1}')
+          find ./docs/installation/ -type f -name "*.md" -exec sed -i.bak "s/$PREVIOUS_RELEASE_VERSION/$LATEST_RELEASE_VERSION/g" {} \;
+
+      - name: Synchronize helm parameters
+        uses: magefile/mage-action@v2
+        with:
+          version: latest
```

```suggestion
          version: latest
          args: sync:chart
```

let's be more explicit, currently we rely on the default target settings

---

done